<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-06-04T13:54:01 IST" name="Default suite" finished-at="2025-06-04T13:54:22 IST" duration-ms="20118">
    <groups>
    </groups>
    <test started-at="2025-06-04T13:54:01 IST" name="Default test" finished-at="2025-06-04T13:54:22 IST" duration-ms="20118">
      <class name="com_powderRoom_LoginTest.UserLoginTest">
        <test-method is-config="true" signature="loadConfig()[pri:0, instance:com_powderRoom_LoginTest.UserLoginTest@21282ed8]" started-at="2025-06-04T13:53:59 IST" name="loadConfig" finished-at="2025-06-04T13:54:01 IST" duration-ms="2785" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loadConfig -->
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com_powderRoom_LoginTest.UserLoginTest@21282ed8]" started-at="2025-06-04T13:54:02 IST" name="setup" finished-at="2025-06-04T13:54:16 IST" duration-ms="14844" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Android]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyLoginDisplayTest()[pri:0, instance:com_powderRoom_LoginTest.UserLoginTest@21282ed8]" started-at="2025-06-04T13:54:16 IST" name="verifyLoginDisplayTest" finished-at="2025-06-04T13:54:20 IST" duration-ms="3950" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1067)
at org.testng.Assert.assertEquals(Assert.java:1031)
at org.testng.Assert.assertEquals(Assert.java:1077)
at com_powderRoom_LoginTest.UserLoginTest.verifyLoginDisplayTest(UserLoginTest.java:34)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginDisplayTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com_powderRoom_LoginTest.UserLoginTest@21282ed8]" started-at="2025-06-04T13:54:21 IST" name="tearDown" finished-at="2025-06-04T13:54:22 IST" duration-ms="687" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="stopServer()[pri:0, instance:com_powderRoom_LoginTest.UserLoginTest@21282ed8]" started-at="2025-06-04T13:54:22 IST" name="stopServer" finished-at="2025-06-04T13:54:22 IST" duration-ms="69" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- stopServer -->
      </class> <!-- com_powderRoom_LoginTest.UserLoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
